client/views/edit/edit.js

```diff
@@ -0,0 +1,72 @@
+import { Instances } from '/lib/common.js';
+
+function showEditAdvError(reason) {
+  if (typeof currentError !== 'undefined') {
+    Blaze.remove(currentError);
+  }
+  const parentNode = $('.formcontainer')[0];
+  const nextNode = $('.inputcontainer')[0];
+  currentError = Blaze.renderWithData(Template.form_error, reason, parentNode, nextNode);
+}
+
+Template.edit.onRendered(() => {
+  // When the template is rendered, set the document title
+  $('.formcontainer').hide().fadeIn(400);
+  $('#darker').hide().fadeIn(400);
+});
+
+Template.edit.helpers({
+  isSelected(property, value) {
+    return Template.instance().data.table[property] === value ? 'selected' : '';
+  },
+});
+
+Template.edit.events({
+  'click .editsubmitbutton': function (event, template) {
+    const table = Template.instance().data.table;
+    const threshold = $('select[name=threshold]')[0].value;
```

Comment:
This can be simplified to `$('select[name=threshold]').val();`. Instead of using the DOM API, it uses the jQuery [val function](http://api.jquery.com/val/). This is slightly better because it'll give you `undefined` instead of an index error if the selector doesn't find anything. This is a super minor point, but it's good to know about that function.

Also, this code is assuming the existence of a `$` global. That might not always be the case. There are two options: either import jQuery as `$`, or use `Template.$` (documented [here](http://blazejs.org/api/templates.html#Blaze-TemplateInstance-\$)). I'd prefer the second option because you don't need to import anything additional and your selector is automatically scoped to the template instance, which might give you a slight performance boost because it's only searching the template for the selector rather than the whole doc.